doc/release-process.md

```diff
@@ -1,168 +1,210 @@
-Release Process
-====================
+# Release Process
 
 ## Branch updates
 
 ### Before every release candidate
-
-* Update translations (ping wumpus on IRC) see [translation_process.md](https://github.com/bitcoin/bitcoin/blob/master/doc/translation_process.md#synchronising-translations).
```

Comment:
back ticks around _translation_process.md_ ?

Reply:
ACK, applied change in 5a797b4